.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
 
     strategy:
       matrix:
-        otp: ['24', '25', '26', '27']
+        otp: ['25', '26', '27']
```

Comment:
Shall we also update https://github.com/inaka/katana-code/blob/main/rebar.config#L6?

Reply:
Yes!